### PR TITLE
Added Bash and Shell Scripting Course in Hindi.

### PR DESCRIPTION
**Pull request by xB0t**
## What does this PR do?
Add resource(s)

## For resources
Bash and Shell Scripting Course in Hindi.

### Why is this valuable (or not)?
Bash and Shell Scripting Course in Hindi is valuable because native people can understand the course.

### How do we know it's really free?
Open the link, Playlist will be available.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a Course, Online on YouTube.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by shrn01 — COMMENTED** [review]
Looks like you have taken an older branch that has added unintended changes. Please take a fresh fork of `main` and update the PR

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -104,6 +104,7 @@
 ### Bash and Shell
 
 * [Bash Basic Commands](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
+* [Bash scripting tutorial for beginners | learn shell scripting in one video in Hindi English](https://www.youtube.com/watch?v=9T2nEXlLy9o&list=PLxLRoXCDIalcosmDOQizh31EIHEK1njfO&index=1&ab_channel=FortifySolutions) - Fortify Solutions (Savita Dadhwal, Soniya)
```
Comment (shrn01) [inline courses/free-courses-hi.md:107]: Please change this link to playlist link

Comment (xB0t) [reply courses/free-courses-hi.md]: Done, Thanks.

**Comment by eshellman** [comment]
remove `.github/workflows/check-urls.yml` from the PR

**Review by xB0t — COMMENTED** [review]
Deleted check-urls.yml file

**Review by shrn01 — COMMENTED** [review]
You have deleted the file, please remove the file from the commit by doing git restore and update the PR so that this change is not there in the PR

**Pull request closed without merging**